src/User/Tests/Infra/Security/SecurityUserCheckerTest.php

```diff
@@ -16,49 +16,44 @@
 
 final class SecurityUserCheckerTest extends TestCase
 {
-    public function testPreAuth(): void
+    public function testPreAuthDisabledUser(): void
     {
+        $this->expectException(DisabledException::class);
```

Comment:
in general i put expectException as late as possible, not causing false positives if i.e. User::__construct (below) throws such an exception